### DISCUSSION TITLE
How can i sign in as a super admin in my local system.

### DISCUSSION THREAD

**Discussion by amitpanwar789** [learningequality/kolibri#9899] [Q&A] [answered]
Hi team,
I want to know, how can I sign in as a super admin or have resource management permission.

**Comment by MisRob** [comment]
Hello,

typically the first account you create when setting up Kolibri is a super admin account. Please see https://kolibri.readthedocs.io/en/latest/manage/permissions.html#permission-to-manage-channels-and-resources to find out more about resources management permissions.

**Comment by MisRob** [comment]
Apologies for the late reply, in the past, we weren't using GitHub Discussion very actively and missed the message.

**Comment by amitpanwar789** [comment]
@MisRob is there any communication channel for kolibri discussion like slack?

**Reply by MisRob** [reply]
@amitpanwar789 This is the right place for posts like your message as well as other categories (you can see new "Categories"). We just didn't use it much in the past. However, recently we made a decision to use GitHub Discussion as the main platform for the contributor community. The whole forum has just been prepared for it and we also have some new mechanisms in place to check on notifications systematically.

**Reply by MisRob** [reply]
^ I made that comment until the assumption you were interested specifically in Kolibri as a developer rather than as a user but now I realize it's not clear from the original post. Just in case it's the latter, you can have a look at [the Kolibri User Guide](https://kolibri.readthedocs.io/en/latest/index.html) or visit [Learning Equality Community Forum](https://community.learningequality.org/) for more general discussions about our products.

**Comment by amitpanwar789** [comment]
Hello again, @MisRob  I apologize for bothering you again, but, When I set up Kolibri as a developer, the first account that was created was a learner account and it shows the 'Learn' section only.

**Reply by jredrejo** [reply, marked as answer]
hello @amitpanwar789 , if you set up kolibri as explained here: https://kolibri.readthedocs.io/en/latest/install/initial_setup.html the account you create is the superadmin.
In case you're using the code from the develop branch of kolibri, the setup wizard has several new options in this initial setup, be sure not to select "On my own" in the first screen, then, selecting the default options you'll create a superadmin account.

In the worst case, if you don't remember the superadmin credentials, you can always create a new superadmin  user following the instructions at https://kolibri.readthedocs.io/en/latest/manage/command_line.html#create-a-new-super-admin

Regards

**Comment by amitpanwar789** [comment]
thanks you for your time @jredrejo , I get this .
